lld/wasm/SyntheticSections.cpp

```diff
@@ -16,6 +16,7 @@
 #include "InputElement.h"
 #include "OutputSegment.h"
 #include "SymbolTable.h"
+#include "llvm/BinaryFormat/Wasm.h"
```

Comment:
Is this strictly needed?  Or did you just notice we were not included it directly?

Reply:
We were already getting it via:
SymbolTable.h --> Symbols.h --> Config.h --> llvm/BinaryFormat/Wasm.h

Reply:
I can remove it if you want, but we use it a lot in this file so it's confusing to get it from such an indirect route.

Reply:
It seems reasonable / good to include what you use.